tests/lib/cli-engine.js

```diff
@@ -712,6 +712,25 @@ describe("CLIEngine", function() {
             assert.equal(report.results[0].messages.length, 0);
         });
 
+        it("should not fail if an ignored file cannot be resolved", function() {
+
+            var fakeFS = leche.fake(fs),
+                LocalCLIEngine = proxyquire("../../lib/cli-engine", {
+                    fs: fakeFS
+                });
+
+            fakeFS.realpathSync = function() {
+                throw new Error("this error should not happen");
+            };
+
+            engine = new LocalCLIEngine({
+                ignorePattern: "tests"
+            });
+
+            engine.executeOnFiles(["tests/fixtures/single-quoted.js"]);
```

Comment:
Ah, yeah sure thing. I was just letting mocha capture a thrown error as indicative of a failing test, but I'll drop in `assert.doesNotThrow()` to make it explicit.